pages/common/sk.md

```diff
@@ -0,0 +1,21 @@
+# sk
+
+> Skim is a fuzzy finder written in rust.
+> Similar to `fzf`.
+> More information: <https://github.com/lotabout/skim>.
+
+- Start finder on all files from arbitrary locations:
+
+`find {{path/to/search}} -type f | sk`
+
+- Start finder on running processes:
```

Comment:
Why is the tool suddenly called finder? Shouldn't it be
```suggestion
- Start skim on running processes:
```
?

Reply:
its a fuzzy finder .. thats why .. called finder here :)

Reply:
But it sounds a bit like `finder` is a name here. Why not just saying `skim`?

Reply:
I'd say that its better to use the product name rather than something that could cause confusion.